From dfc3ddbb3bfac6bc1acb3be1c8ca74b0d920063f Mon Sep 17 00:00:00 2001
From: Josh Wu <josh.wu@atmel.com>
Date: Mon, 24 Sep 2012 08:06:30 +0800
Subject: [PATCH 057/138] mtd: atmel_nand: fix the compile error which miss
 label "err_ecc_ioremap"

Add err_ecc_ioremap label to fix following errors:
---8<---
  CC      drivers/mtd/nand/atmel_nand.o
drivers/mtd/nand/atmel_nand.c: In function 'atmel_nand_probe':
drivers/mtd/nand/atmel_nand.c:1423: error: label 'err_ecc_ioremap' used but not defined
make[3]: *** [drivers/mtd/nand/atmel_nand.o] Error 1
make[2]: *** [drivers/mtd/nand] Error 2
make[1]: *** [drivers/mtd] Error 2
make: *** [drivers] Error 2
--->8---

This error was introduced in b654a9a46fc2100b318050483f8c5b5d0f187303 (mtd: atmel nand: fix gpio missing request)

Signed-off-by: Josh Wu <josh.wu@atmel.com>
Signed-off-by: Jean-Christophe PLAGNIOL-VILLARD <plagnioj@jcrosoft.com>
---
 drivers/mtd/nand/atmel_nand.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/mtd/nand/atmel_nand.c b/drivers/mtd/nand/atmel_nand.c
index edbbf6a..c5679ae 100644
--- a/drivers/mtd/nand/atmel_nand.c
+++ b/drivers/mtd/nand/atmel_nand.c
@@ -1660,6 +1660,8 @@ err_scan_tail:
 		pmecc_writel(host->ecc, CTRL, PMECC_CTRL_DISABLE);
 		pmecc_data_free(host);
 	}
+
+err_ecc_ioremap:
 	if (host->ecc)
 		iounmap(host->ecc);
 	if (host->pmerrloc_base)
-- 
1.7.10.4

